.idea/runConfigurations/Trace_debugger_Remote.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Should we add some documentation for this file?
Something like: "This file set-ups a remote debug configuration to add a possibility to debug Lincheck code when its JVM agent is attached to another application."